src/main/java/org/zalando/nakadi/service/timeline/TimelineService.java

```diff
@@ -227,6 +232,17 @@ public void delete(final String eventTypeName, final String timelineId, final Cl
         }
     }
 
+    public Multimap<TopicRepository, String> deleteAllTimelinesForEventType(final String eventTypeName)
+            throws TopicDeletionException, TimelineException, NotFoundException {
+        LOG.info("Deleting all timelines for event type {}", eventTypeName);
+        final Multimap<TopicRepository, String> topicsToDelete = ArrayListMultimap.create();
+        for (final Timeline timeline : getTimelines(eventTypeName)) {
```

Comment:
@lmontrieux I think you should use getActiveTimelinesOrdered instead of getTimelines